server/src/main/java/io/crate/metadata/Functions.java

```diff
@@ -331,12 +335,50 @@ public FunctionImplementation getQualified(Signature signature,
     private static UnsupportedOperationException raiseUnknownFunction(@Nullable String suppliedSchema,
                                                                       String name,
                                                                       List<? extends Symbol> arguments) {
-        StringJoiner joiner = new StringJoiner(", ");
-        for (var arg : arguments) {
-            joiner.add(arg.valueType().toString());
+        String functionString = formatPossibleOperators(suppliedSchema, name, arguments);
```

Comment:
Did you consider to create a `Function` symbol to be able to delegate the formatting to the `toString` implementation of `Function`?

Reply:
Did consider, but forget somehow. Pushed a fixup, thanks for the heads up.